export_test.go

```diff
@@ -0,0 +1,20 @@
+package sqlair
+
+import (
+	"database/sql"
+	"sync"
+
+	"github.com/google/uuid"
```

Comment:
We don't need that kind of UUID for this purpose. Let's just have a monotonic integer atomically incremeneted.

Reply:
Changed.